guillotina/interfaces/json.py

```diff
@@ -8,10 +8,10 @@
 class IResourceSerializeToJson(Interface):
     """Adapter to serialize a Resource into a JSON object."""
 
-    def __init__(self, context, request):
+    def __init__(context, request):
```

Comment:
Whatt ?? no self ??

Reply:
This is what way interfaces are supposed to be defined. zope's mypy supports complains about it.